ruby/all-but-the-first-element-from-array.md

```diff
@@ -21,6 +21,35 @@ rest_of_list = list
 # => [2,3,4]
 ```
 
+## Parallel assignment
+
+We can also use parallel assignment to get the first element of the list. This
+does not mutate the original array.
+
+```ruby
+head, *tail = list
+#=> [1,2,3,4]
+head
+#=> 1
+tail
+#=> [2,3,4]
+
+list # is not modified
+#=> [1,2,3,4]
+```
+
+This has the same effect as `#drop` on the tail when applied to the empty array.
```

Comment:
@JoelQ You're right. I've updated the wording. Please, let me know if it needs any further changes.